File: 01-02-2024/par-ou-impar/ParOuImpar.java
import java.util.Scanner;

public class ParOuImpar {

    public static void main(String[] args) {
        try (Scanner scanner = new Scanner(System.in)) {
            System.out.println("Informe um inteiro: ");
            
            final int a = scanner.nextInt();
            final String state;

            if (a % 2 == 0) {
                state = "par";
            } else {
                state = "ímpar";
            }

            System.out.format("%s é %s.", a, state).println();
        }
    }

}

File: 01-02-2024/par-ou-impar/par_ou_impar.c
# include <stdio.h>

int main() {
    int a;

    printf("Informe um inteiro: ");
    scanf("%d", &a);

    if (a % 2) {
        printf("%d é par.\n", a);
    } else {
        printf("%d é ímpar.\n", a);
    }

    return 0;
}